assets/tests/components/notificationCard.test.tsx

```diff
@@ -263,7 +272,7 @@ describe("NotificationCard", () => {
         />
       </RoutesProvider>
     )
-    expect(baseElement).toMatchSnapshot()
+    await waitFor(() => expect(baseElement).toMatchSnapshot())
   })
```

Comment:
if https://github.com/mbta/skate/pull/2835#discussion_r1786716063 fixes this test failiure, I think we should try removing the async here as well, as I don't _think_ it's needed?

Reply:
Yes! Fab, thank you